datafusion/core/tests/sql/subqueries.rs

```diff
@@ -252,15 +252,8 @@ async fn tpch_q17_correlated() -> Result<()> {
         );"#;
 
     // assert plan
-    let plan = ctx
-        .create_logical_plan(sql)
-        .map_err(|e| format!("{:?} at {}", e, "error"))
```

Comment:
These map_err were only adding a context of `"error"` so I just opted to remove them :sweat_smile: